reference/orders.v2.oas2.yml

```diff
@@ -3543,43 +3551,47 @@ components:
           example: 1
           type: integer
         order_id:
-          description: 'The unique numeric identifier of the order to which the tax was applied. NOTE: Not included if the store was using the automatic tax feature.'
+          description: |- 
+            The unique numeric identifier of the order to which the tax was applied. NOTE: Not included if the store was using the automatic tax feature.
           example: 129
           type: integer
         order_address_id:
-          description: 'The unique numeric identifier of the order address object associated with the order. NOTE: Not included if the store was using the automatic tax feature.'
+          description: |-
+            The unique numeric identifier of the order address object associated with the order. NOTE: Not included if the store was using the automatic tax feature.
           example: 29
           type: integer
         tax_rate_id:
           description: The unique numeric identifier of the tax rate.
           example: 1
           type: integer
         tax_class_id:
-          description: 'The unique numeric identifier of the tax class object. NOTE: Will be 0 if automatic tax was enabled, or if the default tax class was used.'
+          description: |- 
+            The unique numeric identifier of the tax class object. NOTE: Will be 0 if automatic tax was enabled, or if the default tax class was used.
           example: 0
           type: integer
         name:
           description: The name of the tax class object.
-          example: '"State Tax"'
+          example: "State Tax"
           type: string
         class:
-          description: 'The name of the type of tax that was applied. NOTE: will be "Automatic Tax" if automatic tax was enabled. Will be "API Tax Override" if the order was created with V2 Orders API.'
+          description: |-
+            The name of the type of tax that was applied. NOTE: will be "Automatic Tax" if automatic tax was enabled. Will be "API Tax Override" if the order was created with V2 Orders API.
```

Comment:
```suggestion
            The name of the type of tax that was applied. NOTE: It will be "Automatic Tax" if automatic tax was enabled. It will be "API Tax Override" if the order was created with V2 Orders API.
```